teuthology/repo_utils.py

```diff
@@ -97,20 +97,25 @@ def enforce_repo_state(repo_url, dest_path, branch, remove_on_error=True):
         raise
 
 
-def clone_repo(repo_url, dest_path, branch):
+def clone_repo(repo_url, dest_path, branch, shallow=True):
```

Comment:
yeah.  and nothing appears to use fetch_branch but tests.  shrug.  It's likely fine.